src/tokens/TokenDetails.tsx

```diff
@@ -76,12 +78,19 @@ export default function TokenDetailsScreen({ route }: Props) {
             {token.name}
           </Text>
         </View>
-        <TokenDisplay
-          amount={token.balance}
-          tokenId={tokenId}
-          style={styles.balance}
-          testID="TokenDetails/Balance"
-        />
+        {token?.priceUsd !== null ? (
+          <TokenDisplay
+            amount={1}
+            tokenId={tokenId}
+            style={styles.assetValue}
+            testID="TokenDetails/AssetValue"
```

Comment:
Looks like e2e test is failing as it looks for `TokenDetails/Balance`

Reply:
thanks, fixed